vectordraw.css

```diff
@@ -0,0 +1,52 @@
+body {
+    margin: 10px;
+}
+
+.jxgboard {
+    float: left;
+    border: 2px solid #1f628d;
+    /* background-size: contain; */
+    /* background-repeat: no-repeat; */
+    /* background-position: center; */
+    /* background-size: 80% auto; */
+}
+
+.menu {
+    float: left;
+    width: 170px;
+    margin-left: 15px;
+}
+
+.menu select {
+    width: 160px;
+    margin-bottom: 8px;
+}
+
+.menu button {
+    background-color: #3498db;
+    border-radius: 5px;
+    box-shadow: 0 1px 3px #666;
+    font-family: Arial, sans-serif;
+    color: #fff;
+    font-size: 15px;
+    padding: 5px 10px 5px 10px;
+    margin: 4px 0;
+    border: 2px solid #1f628d;
+    text-decoration: none;
+    width: 160px;
+}
+
+.menu button:hover {
+    background: #3cb0fd;
+    background-image: -webkit-linear-gradient(top, #3cb0fd, #3498db);
+    background-image: -moz-linear-gradient(top, #3cb0fd, #3498db);
+    background-image: -ms-linear-gradient(top, #3cb0fd, #3498db);
+    background-image: -o-linear-gradient(top, #3cb0fd, #3498db);
+    background-image: linear-gradient(to bottom, #3cb0fd, #3498db);
+    text-decoration: none;
+}
+
+.menu button.undo,
+.menu button.redo {
+    width: 78px;
+}
```

Comment:
@mtyaka nit: no newline at the end of file. We don't have style checks in this repo, but it would be nice to still practice them :)